instruments/keithley/keithley195.py

```diff
@@ -259,8 +259,7 @@ def measure(self, mode=None):
             current_mode = self.mode
             if mode != current_mode:
                 self.mode = mode
-                if not self._testing:
-                    time.sleep(2)  # Gives the instrument a moment to settle
+                time.sleep(2)  # Gives the instrument a moment to settle
```

Comment:
As before, I removed `if not self._testing` and am mocking the `time.sleep` in the test now. Also asserting that it is called with the argument of 2 (seconds).